core/txbuilder/constraint.go

```diff
@@ -41,7 +41,8 @@ func (t timeConstraint) code() []byte {
 		}
 		builder.AddOp(vm.OP_MAXTIME).AddInt64(int64(t.maxTimeMS)).AddOp(vm.OP_LESSTHANOREQUAL)
 	}
-	return builder.Program
+	prog, _ := builder.Build()
```

Comment:
Does it help that I added a promise to the contract of `Builder.Build` saying that errors are impossible if you never call `AddJump` or `AddJumpIf`?